pkg/service/mocktest/mocktest.go

```diff
@@ -21,13 +21,13 @@ import (
 var Emoji = "\U0001F430" + " Keploy:"
 
 type mockTester struct {
-	logger *zap.Logger
+	Logger *zap.Logger
```

Comment:
If logger is not used externally, it could be unexported.

Reply:
done